packages/neo-one-node-bin/src/utils/getConfiguration.ts

```diff
@@ -67,6 +69,8 @@ const getFreshConfig = (): NodeBinEnvironment => {
     settingsEnvironment: config.settings,
     monitorEnvironment: config.monitor,
     options: config.options === undefined ? {} : config.options,
+    dumpChainFile: config.dumpChainFile,
```

Comment:
We should be uploading this chain file somewhere.

Reply:
I just submitted a change so that the dumpChainFile would stop crashing because of memory overflows. I thought it was an issue of the writestream not draining, but actually it was the 
```
const blocks = await Promise.all(chunk.map(async (index) => blockchain.block.get({ hashOrIndex: index })));
```
line creating too big an array. I reduced this to only poll 1000 blocks at a time instead of 10000 which solved the problem and had no impact on the processing speed.

Should I just configure them to upload alongside the backup?